### PR TITLE
clamav - improve weekly full-scan tremendously

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Comment by szaimen** [comment]
This is ready 👍
This is now tremendously improved by only scanning all files the first time it runs and afterwards only scanning all the files that were changed during the last week which should speed up all successor runs by a lot!

File: apps/clamav.sh [hunk 1]
```diff
@@ -252,41 +253,51 @@ cat << CLAMAV_REPORT > "$SCRIPTS"/clamav-fullscan.sh
...
+
+if [ -n "\$FULLSCAN_DONE" ]
+then
+    FIND_OPTS+=(-ctime -7)
+    TIMEOUT=(timeout 12h)
+fi
+
+find / "\${FIND_OPTS[@]}" | tee /tmp/scanlist
+
+"\${TIMEOUT[@]}" \
```
[the remark this answers is not in the document]
Comment (szaimen) [reply apps/clamav.sh]: done

**Comment by szaimen** [comment]
This should be it 👍

**Comment by enoch85** [comment]
Is it possible to add a check to see if the scan is run?

**Comment by szaimen** [comment]
One good thing about this PR is that it will log all scanned file to this file: `/var/log/nextcloud/clamav-fullscan.log`. So you can simply run a `tail -f /var/log/nextcloud/clamav-fullscan.log` to monitor the currently scanned files during a scan. This wasn't possible before and hence very hard to debug

**Comment by enoch85** [comment]
> @enoch85 is this what you wanted?

Yup!

**Pull request merged**